massa-models/src/block.rs

```diff
@@ -933,7 +947,7 @@ mod test {
         // create block header
         let orig_header = BlockHeader::new_wrapped(
             BlockHeader {
-                slot: Slot::new(1, 1),
+                slot: Slot::new(0, 1),
```

Comment:
Does this change correct an error? my current understanding is that a genesis block (i.e. the first block in a thread) has period of 0

Reply:
This makes the test correct. Before the test wasn't testing the right behavior.